src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -50,6 +53,9 @@ internal abstract class VisualStudioWorkspaceImpl : VisualStudioWorkspace
 
         private readonly ForegroundThreadAffinitizedObject _foregroundObject = new ForegroundThreadAffinitizedObject();
 
+        private PackageInstallerService _packageInstallerService;
```

Comment:
Thse get assigned in InitializeStandardVisualStudioWorkspace, not the constructor.  So they can't be readonly (unfortunatley).